app/components/Pagination.tsx

```diff
@@ -0,0 +1,64 @@
+import { tw, tx } from '@helpwave/common/twind/index'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import ChevronLast from '@helpwave/common/icons/ChevronLast'
+import ChevronLeft from '@helpwave/common/icons/ChevronLeft'
+import ChevronFirst from '@helpwave/common/icons/ChevronFirst'
+import ChevronRight from '@helpwave/common/icons/ChevronRight'
+
+type PageIndicatorTranslation = {
+  of: string
+}
+const defaultOrganizationMemberListTranslations: Record<Languages, PageIndicatorTranslation> = {
+  en: {
+    of: 'of'
+  },
+  de: {
+    of: 'von'
+  }
+}
+
+export type PageIndicatorProps = {
+  page: number, // starts with 0
+  numberOfPages: number,
+  onPageChanged: (page: number) => void
+}
+
+export const Pagination = ({
+  language,
+  page,
+  numberOfPages,
+  onPageChanged
+}: PropsWithLanguage<PageIndicatorTranslation, PageIndicatorProps>) => {
+  const translation = useTranslation(language, defaultOrganizationMemberListTranslations)
+
+  const changePage = (page:number) => {
+    onPageChanged(page)
+  }
+
+  const onFirstPage = page === 0
+  const onLastPage = page === numberOfPages - 1
+
+  return (
+    <div className={tw('flex flex-row')}>
+      <button onClick={() => changePage(0)} disabled={onFirstPage}>
+        <ChevronFirst className={tx({ 'opacity-30': onFirstPage })}/>
+      </button>
+      <button onClick={() => changePage(page - 1)} disabled={onFirstPage}>
+        <ChevronLeft className={tx({ 'opacity-30': onFirstPage })}/>
```

Comment:
the icon uses `currentColor` internally, meaning you can recolor it by changing the text color. That is the preferred approach for this: `tw(onFirstPage ? 'text-...' : 'text-...')`